homeassistant/components/zwave_js/translations/en.json

```diff
@@ -51,6 +51,16 @@
             }
         }
     },
+    "device_automation": {
+        "trigger_type": {
+            "event.notification.entry_control": "Sent a notification (Entry Control CC)",
+            "event.notification.notification": "Sent a notification (Notification CC)",
+            "event.value_notification.basic": "Basic CC event on {subtype}",
```

Comment:
This is not very informative to the user. Is there a way we can clarify this further ?

Reply:
Like I wrote on discord, I don't think this is possible, since it depends on how the device is programmed and we'd have to tell a longer story to explain that in-depth. Entry Control notifications are specific to the Entry Control CC, which can be an armed or disarmed alarm, a gate that was opened, etc.
Basic events are a concept of zwave-js to capture stuff that nodes send as Basic Sets (if they are not mapped to a report) and should really have used a different CC for.
Notification CC just captures anything else, from fire alarm to a locked/unlocked door.

Maybe a bit less repetitive:
```
        "trigger_type": {
            "event.notification.entry_control": "Sent an Entry Control notification",
            "event.notification.notification": "Sent a notification",
            "event.value_notification.basic": "Basic CC event on {subtype}",
```